lib/options-handler.js

```diff
@@ -279,7 +279,15 @@ export class ClientOptionsHandler {
             'postProcess', 'demanglerType', 'isSemVer']);
         const copiedCompilers = JSON.parse(JSON.stringify(compilers));
         let semverGroups = {};
+        // Reset the supportsExecute flag in case critical compilers change
+        _.each(this.options.languages, (lang) => {
+            lang.supportsExecute = false;
+        });
+
         _.each(copiedCompilers, (compiler, compilersKey) => {
+            if (compiler.supportsExecute) {
+                this.options.languages[compiler.lang].supportsExecute = true;
+            }
```

Comment:
Tiny nit: ss there any reason for using _.each for an array on the backend over Array.forEach?

Reply:
Old habits I guess! @RubenRBS or @supergrecko I'd be happy for you to come back and fix this. Right now I'm trying to get through all the PRs outstanding :)